### PR TITLE
report builder minor fixes

### PR DESCRIPTION
**Pull request by czue**
started poking around in report builder today and couldn't help myself.

this addresses https://manage.dimagi.com/default.asp?282631 with this error message:

image

It also should fix these sentry errors:

https://sentry.io/dimagi/commcarehq/issues/518359740/ https://sentry.io/dimagi/commcarehq/issues/558242566/
https://sentry.io/dimagi/commcarehq/issues/570325581

### CODE REVIEW

File: corehq/apps/userreports/reports/builder/forms.py [hunk 1]
```diff
@@ -1291,27 +1291,28 @@ class ConfigureTableReportForm(ConfigureListReportForm):
...
-                    "x_axis_column": get_agged_columns()[0]['column_id'] if get_agged_columns() else '',
+                    "x_axis_column": agged_columns[0]['column_id'] if agged_columns else '',
                     # TODO: Possibly use more columns?
                     "y_axis_columns": [
-                        {"column_id": c["column_id"], "display": c["display"]} for c in get_non_agged_columns()
+                        {"column_id": c["column_id"], "display": c["display"]} for c in non_agged_columns
                     ],
                 }]
             elif self.cleaned_data['chart'] == "pie":
+                if not agged_columns or not non_agged_columns:
```
Comment (mkangia) [inline corehq/apps/userreports/reports/builder/forms.py]: you already checked for `non_agged_columns` above so this check here should not be needed?

**Review by mkangia — COMMENTED** [review]
Not completely sure about the change to the user_id.
But rest looks good regarding the error message.

**Comment by mkangia** [comment]
> couldn't help myself.

Could not help yourself so indeed just helped others :)

File: corehq/apps/userreports/reports/builder/forms.py (hunk 1 above)
Comment (czue) [reply corehq/apps/userreports/reports/builder/forms.py]: oops good call. https://github.com/dimagi/commcare-hq/pull/21773/commits/2c84823b1ba7d47d441d2654e1ee3bd29973bd25

**Comment by czue** [comment]
@mkangia / anyone else let me know if any other feedback on this guy

**Review by calellowitz — APPROVED** [review]

**Comment by czue** [comment]
> Not completely sure about the change to the user_id.

@mkangia definitely though am not sure I understand the question. could you try elaborating a bit or commenting in the code where you have it?

**Pull request merged**